policyengine_us/parameters/gov/states/ky/tax/income/exemptions/aged/age.yaml

```diff
@@ -0,0 +1,21 @@
+description: Kentucky provides a homestead exemption of this value for each aged over 65
+values:
+  2019-01-01: 39_300
+  2020-01-01: 39_300
+  2021-01-01: 40_500
+  2022-01-01: 40_500
+  2023-01-01: 46_350
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Kentucky homestead exemption for each aged over 65"
+  reference:
+  - title: Kentucky Homestead Exemption
```

Comment:
need one legal code reference

Reply:
[law](https://casetext.com/statute/kentucky-revised-statutes/title-11-revenue-and-taxation/chapter-132-levy-and-assessment-of-property-taxes/section-132810-homestead-exemption-application-qualification)

Reply:
[tax form](https://revenue.ky.gov/Forms/740%20Packet%20Instructions%205-9-23.pdf)